pebble_tool/sdk/manager.py

```diff
@@ -115,6 +115,14 @@ def _install_from_handle(self, f):
             subprocess.check_call([os.path.join(virtualenv_path, "bin", "python"), "-m", "pip", "install", "-r",
                                    os.path.join(path, "sdk-core", "requirements.txt")],
                                   env={'PYTHONHOME': virtualenv_path})
+            package_json = os.path.join(path, "sdk-core", "package.json")
+            if os.path.exists(package_json) and self._js_prompt():
```

Comment:
what happens if a users says "no" here at installation time? How will they ever be able to install the node dependencies later on? I couldn't see a second reference to `_js_prompt()` which indicates another place where we check and install.
Ideally, we should check and offer when users build or create a `rocky` project.